examples/nextjs-with-typescript/certificates/localhost-key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
NOTE: These are locally created keys for testing https servers locally (a pre-requisite for some DRM test cases). No concerns with these being pushed to remote repo.